diff --git a/bootstrap/app.php b/bootstrap/app.php
index 5cedded..08cc5e5 100644
--- a/bootstrap/app.php
+++ b/bootstrap/app.php
@@ -1,9 +1,12 @@
 <?php
 
 use Illuminate\Foundation\Application;
+use Bugsnag\BugsnagLaravel\OomBootstrapper;
 use Illuminate\Foundation\Configuration\Exceptions;
 use Illuminate\Foundation\Configuration\Middleware;
 
+(new OomBootstrapper())->bootstrap();
+
 return Application::configure(basePath: dirname(__DIR__))
     ->withRouting(
         web: __DIR__.'/../routes/web.php',
