diff --git a/app/Http/Kernel.php b/app/Http/Kernel.php
index 17b4866..d1dcc9e 100644
--- a/app/Http/Kernel.php
+++ b/app/Http/Kernel.php
@@ -67,4 +67,16 @@ class Kernel extends HttpKernel
         'hanMidEx' => \App\Http\Middleware\HandledMiddlewareEx::class,
         'hanMidErr' => \App\Http\Middleware\HandledMiddlewareErr::class,
     ];
+
+    protected function bootstrappers()
+    {
+        if (!getenv('BUGSNAG_REGISTER_OOM_BOOTSTRAPPER')) {
+            return parent::bootstrappers();
+        }
+
+        return array_merge(
+            [\Bugsnag\BugsnagLaravel\OomBootstrapper::class],
+            parent::bootstrappers(),
+        );
+    }
 }
