diff --git a/config/app.php b/config/app.php
index 1941d7c..039b0d4 100644
--- a/config/app.php
+++ b/config/app.php
@@ -167,6 +167,7 @@
         /*
          * Package Service Providers...
          */
+        Bugsnag\BugsnagLaravel\BugsnagServiceProvider::class,
 
         /*
          * Application Service Providers...
@@ -191,7 +192,7 @@
     */
 
     'aliases' => Facade::defaultAliases()->merge([
-        // ...
+        'Bugsnag' => Bugsnag\BugsnagLaravel\Facades\Bugsnag::class,
     ])->toArray(),
 
 ];
